src/Compatibility/Microsoft.DotNet.ApiCompatibility/Resources.resx

```diff
@@ -213,4 +213,10 @@
   <data name="EnumTypesMustMatch" xml:space="preserve">
     <value>Underlying type of enum '{0}' changed from '{1}' to '{2}'.</value>
   </data>
+  <data name="CannotAddVirtualModifier" xml:space="preserve">
+    <value>Cannot add virtual modifier to member '{0}'.</value>
+  </data>
+  <data name="CannotRemoveVirtualModifier" xml:space="preserve">
+    <value>Cannot remove virtual modifier from member '{0}'.</value>
```

Comment:
```suggestion
    <value>Cannot make member '{0}' non-virtual.</value>
```

Reply:
While this *is* shorter, I worry that "cannot" and "non-virtual" is bit hard to parse, since it's a double negative.

Reply:
My feedback is not so much about the wording, so feel free to change that as you wish, but more around having the message express the breaking change as opposed to the action that caused the breaking change. For example, the error message above that you added on the previous PR says something around "Underlying type of enum changed from foo to bar" instead of saying: "Cannot add `: short` to enum Foo"

Reply:
Actually I just went to our [official docs for breaking changes](https://docs.microsoft.com/en-us/dotnet/core/compatibility/) to see how we word it over there, and they seem to agree with what you had originally:

![image](https://user-images.githubusercontent.com/13854455/171302812-1b172d13-4609-4f7c-a740-db5bc5c55ba8.png)